src/GovUk.Education.ExploreEducationStatistics.Admin/Models/Api/DataBlockViewModel.cs

```diff
@@ -7,10 +7,12 @@
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Models.Api
 {
-    public class DataBlockViewModel
+    public class DataBlockViewModel : IContentBlockViewModel
     {
         public Guid Id { get; set; }
 
+        public List<CommentViewModel> Comments { get; set; }
```

Comment:
Should the Comments field not be a field on IContentBlockViewModel?

Reply:
Oh I see, it's an interface, gotcha.  Maybe there's scope for an abstract base class at some point  in the future.

Reply:
Yes I agree.